
No - this is not an usual gcc-3.4 fix - this is a real bug-fix ...

  - Rene Rebe <rene@rocklinux.org>

--- gimp-2.0.1/app/paint-funcs/paint-funcs.h	2004-03-10 14:13:43.000000000 +0100
+++ gimp-2.0.1-fixed/app/paint-funcs/paint-funcs.h	2004-05-06 00:15:59.000000000 +0200
@@ -356,7 +356,7 @@
 					   gint,
                                            GimpLayerModeEffects,
                                            gboolean *,
-                                           CombinationMode);
+                                           InitialMode);
 
 void  combine_regions                     (PixelRegion *, PixelRegion *,
 					   PixelRegion *, PixelRegion *,
